frontend/src/Commons/Forms/InputModules/FarmInformation/FarmInformation.tsx

```diff
@@ -49,6 +48,7 @@ const FarmInfoComponent: React.FC<InputModuleProps> = ({ farmDetails, updateFarm
       farmInformation.FarmName = values.FarmName;
       farmInformation.Year = values.Year;
       farmInformation.FarmRegion = values.FarmRegion;
+      farmInformation.HasBerries = false; // Creating nmp file return false, loading returns true
```

Comment:
Same deal here, I had it follow the basic structure of .nmp

```json
{
  "farmDetails": {
    "Year": "2024",
    "FarmName": "FarmName",
    "FarmRegion": 21,
    "FarmSubRegion": null,
    "SoilTests": null,
    "TestingMethod": "11",
    "Manure": null,
    "HasSelectedFarmType": true,
    "ImportsManureCompost": true,
    "HasAnimals": false,
    "HasDairyCows": false,
    "HasBeefCows": false,
    "HasPoultry": false,
    "HasMixedLiveStock": false,
    "HasHorticulturalCrops": true,
    "HasBerries": true,
    "LeafTests": null,
    "LeafTestingMethod": "0",
    "UserJourney": 7
  },
 ```

Reply:
@GDamaso for it's FarmRegion I've noticed that it's a number type. Is it equivalent to it's region? like let's say vancouver island is 21 what do you think?

Reply:
Does the .nmp file really use PascalCase for its object values?